docker/build.Dockerfile

```diff
@@ -0,0 +1,76 @@
+# backward compatibility:
+FROM ubuntu:12.04.5
```

Comment:
Does it make sense to still use 12.04 here? It will be EOL in April: https://wiki.ubuntu.com/Releases